docs/platform/conditions/conditions.json

```diff
@@ -0,0 +1,17 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://example.com/object1592241468.json",
+  "title": "Root",
+  "type": "array",
+  "items": {
+    "$id": "#root/conditions/items",
+    "title": "Items",
+    "type": "object",
+    "required": [
+      "triggers",
+      "action",
+      "schedule"
+    ]
+  }
```

Comment:
discussed on slack with @kailashrkk on potentially using `$ref` keyword to reference the separated json files for triggers and actions in this parent conditions json. This would allow us to only need one call to validate_json in control server

https://json-schema.org/understanding-json-schema/structuring.html#extending